docs/src/getting-started/sway-toolchain.md

```diff
@@ -4,12 +4,14 @@ The Sway toolchain consists of several components.
 
 ## Forc (`forc`)
 
-The "Fuel Orchestrator" Forc is our equivalent of Rust's [Cargo](https://doc.rust-lang.org/cargo/). It is the primary entry point for creating, building, testing, and deploying Sway projects. The next pages in this section will introduce how to use Forc.
+The "Fuel Orchestrator" [Forc](https://github.com/FuelLabs/sway/tree/master/forc) is our equivalent of Rust's [Cargo](https://doc.rust-lang.org/cargo/). It is the primary entry point for creating, building, testing, and deploying Sway projects. The next pages in this section will introduce how to use Forc.
 
 ### Sway Language Server (`forc lsp`)
 
-The Sway Language Server `forc lsp` is provided to expose features to IDEs. Currently, only [Visual Studio Code is supported through a plugin](https://marketplace.visualstudio.com/items?itemName=FuelLabs.sway-vscode-plugin). Vim support is forthcoming, though [syntax highlighting is provided](https://github.com/FuelLabs/sway.vim). Note that there is no need to manually run `forc lsp` (the plugin will automatically start it), however, it should be included in your `$PATH` if you are building `forc` from source.
+The Sway Language Server `forc lsp` is provided to expose features to IDEs. Currently, only [Visual Studio Code is supported through a plugin](https://marketplace.visualstudio.com/items?itemName=FuelLabs.sway-vscode-plugin). Vim support is forthcoming, though [syntax highlighting is provided](https://github.com/FuelLabs/sway.vim).
+
+Note that there is no need to manually run `forc lsp` (the plugin will automatically start it), however `forc` should be included in your `$PATH` if you are building `forc` from source.
```

Comment:
Technically it should be in your path either way, it just happens automatically if you use  `cargo install`.

```suggestion
Note that there is no need to manually run `forc lsp` (the plugin will automatically start it), however `forc` must be in your `$PATH`. To check if `forc` is in your `$PATH`, type `forc --help` in your terminal.
```